src/components/skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "./Skeleton.module.css";
+
+interface SkeletonI {
+  width?: number;
+  height?: number;
+  type?: "circle" | "rectangle" | "rounded";
```

Comment:
instead of rounded, I think a borderRadius prop will help more? that way dev can define the curve they want?

Reply:
i think we need same curves throughout the app ( as design perspective) that's why I gave type option only.
(if it's needed we can change later)

Reply:
🤔makes sense, can we create a variable for that in [variables.css](../blob/develop/src/assets/css/variables.css)? makes it easier to change later and this way everyone will be using the same value?